__tests__/app.test.js

```diff
@@ -267,6 +267,51 @@ describe('/api/reviews/:review_id', () => {
 
 });
 
+describe('/api/comments/:comment_id', () => {
+
+    test('DELETE- status:204 - deletes the comment with the given comment_id', () => {
+        return request(app)
+            .delete('/api/comments/6')
+            .expect(204)
+            .then(() => {
+                return request(app)
```

Comment:
This is fine to do - but you can also just not include it, as receiving the 204 is enough to know the delete was successful. It's fine to leave it as is :)